Added entities and jpa connection

d0vy403 opened the pull request.

Commit:
Added entities and jpa connection

Jurkenas commented on src/main/java/lt/codeacademy/restemployee/entity/Department.java:
```diff
@@ -0,0 +1,25 @@
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
```
Nenaudojamas

Jurkenas commented on src/main/java/lt/codeacademy/restemployee/entity/Employee.java:
```diff
@@ -0,0 +1,42 @@
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
```
Nenaudojamas

Jurkenas commented on src/main/java/lt/codeacademy/restemployee/entity/Project.java:
```diff
@@ -0,0 +1,24 @@
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
```
Nenaudojamas

Jurkenas commented on src/main/java/lt/codeacademy/restemployee/RestEmployeeApplication.java:
```diff
@@ -10,4 +20,48 @@ public static void main(String[] args) {
+      System.out.println("=== All Employees ===");
+      employeeRepo.findAll().forEach(System.out::println);
+    };
+  }
```
Nėra service klasių, visa šita logika turėtų būti jose

Commit:
Added controllers for :
Create new employee
Get all employees
Get employee by id
Update employee by id
Delete employee by id
